.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "uwrt_mars_rover_hw_bridge"]
+	path = uwrt_mars_rover_hw_bridge
```

Comment:
@jekov Should it reside here or in a different folder? personally i think this is fine

Reply:
@jetkov ?

Reply:
It would be nice if we cloned it to match the same naming convention as the rest of fw folders
(repo can remain same name)